core/divesitehelpers.cpp

```diff
@@ -22,24 +22,20 @@
 #include <QTimer>
 #include <memory>
 
-void reverseGeoLookup(degrees_t latitude, degrees_t longitude, taxonomy_data *taxonomy)
+/** Performs a REST request to a service returning a JSON object. */
+QJsonObject doAsyncRest(QString url, int msTimeout)
```

Comment:
1) Is this function used outside of this source file? If not, make it `static`, please.

2) Not that it matters, since `QString` has COW & refcounting semantics. However, I would still prefer `const QString &` unless you really need a copy here.

Reply:
REST appears to be an acronym. Therefore, I would suggest `doAsyncREST`.